doesn,t work after update to version

I solve it open my project in xCode and run build to find possible error, so when find error line xcode  suggest add automatically  **if #available** before a version validation and it works. Sorry I close xcode after works and I dont know where the error, but on Android Studio still have the log. I hope it help.


/Users/robertorosa/Documents/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_web_auth-0.2.3/ios/Classes/SwiftFlutterWebAuthPlugin.swift:29:33: error: 'SFAuthenticationError' is only available in iOS 11.0 or newer
                            if case SFAuthenticationError.canceledLogin = err {
                                    ^
    /Users/robertorosa/Documents/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_web_auth-0.2.3/ios/Classes/SwiftFlutterWebAuthPlugin.swift:29:33: note: add 'if #available' version check
                            if case SFAuthenticationError.canceledLogin = err {
                                    ^
    /Users/robertorosa/Documents/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_web_auth-0.2.3/ios/Classes/SwiftFlutterWebAuthPlugin.swift:13:17: note: add @available attribute to enclosing instance method
        public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
                    ^
    /Users/robertorosa/Documents/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_web_auth-0.2.3/ios/Classes/SwiftFlutterWebAuthPlugin.swift:6:14: note: add @available attribute to enclosing class
    public class SwiftFlutterWebAuthPlugin: NSObject, FlutterPlugin {
                 ^
    /Users/robertorosa/Documents/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_web_auth-0.2.3/ios/Classes/SwiftFlutterWebAuthPlugin.swift:18:17: warning: variable 'sessionToKeepAlive' was written to, but never read
                var sessionToKeepAlive: Any? = nil // if we do not keep the session alive, it will get closed immediately while showing the dialog

--- a/ios/Classes/SwiftFlutterWebAuthPlugin.swift
+++ b/ios/Classes/SwiftFlutterWebAuthPlugin.swift
@@ -25,7 +25,9 @@ public class SwiftFlutterWebAuthPlugin: NSObject, FlutterPlugin {
                             result(FlutterError(code: "CANCELED", message: "User canceled login", details: nil))
                             return
                         }
-                    } else {
+                    }
+                    
+                    if #available(iOS 11, *) {
                         if case SFAuthenticationError.canceledLogin = err {
                             result(FlutterError(code: "CANCELED", message: "User canceled login", details: nil))
                             return

